fbw-a380x/src/base/flybywire-aircraft-a380-842/config/a380x/a380-842/aircraft_preset_procedures.xml

```diff
@@ -223,10 +223,14 @@ The StepType also defines if the step is executed in normal mode, expedited mode
             <Condition>(A:CIRCUIT CONNECTION ON:80, Bool)</Condition>
             <Action>80 1 (&gt;K:2:ELECTRICAL_BUS_TO_CIRCUIT_CONNECTION_TOGGLE)</Action>
         </Step>
-        <Step Name="Fuel Trim TK R On" Type="STEP" Delay="500">
+        <Step Name="Fuel Trim TK R On" Type="STEP" Delay="1000">
             <Condition>(A:CIRCUIT CONNECTION ON:81, Bool)</Condition>
             <Action>81 1 (&gt;K:2:ELECTRICAL_BUS_TO_CIRCUIT_CONNECTION_TOGGLE)</Action>
         </Step>
+        <Step Name="NSS DATA TO AVNCS ON" Type="STEP" Delay="1000">
```

Comment:
Isn't this usually always on? I feel like we should fix this in the FLT files or something then